src/Export/ProductDebugService.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export;
+
+use FINDOLOGIC\FinSearch\Export\Errors\ExportErrors;
+use FINDOLOGIC\FinSearch\Utils\Utils;
+use Shopware\Core\Content\Product\ProductEntity;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\EntitySearchResult;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class ProductDebugService extends ProductService
```

Comment:
This could be a real PITA, since we're restructuring the `ProductService` in the near future, therefore relying on it here would result in more work.

Reply:
Leaving this for now, as I am not using functionalities where I expect a lot of changes during variant restructuring.
Would also not be great, to have a lot of duplicated code in those two services